contracts/Pool.sol

```diff
@@ -121,7 +123,8 @@ contract LiquidityPool is ReentrancyGuard {
 
     //User withdraws their percentage of the pool and rewards
     function withdrawAll() public returns (bool) {
-        //User total Balance of the pool after the expiration of the pool
+        require(poolOwnerBalance[msg.sender] >= 0, "Insufficient funds");
```

Comment:
`> 0` (because 0 would then revert in `withdraw()`)

Reply:
Thanks I fixed it.